muni_portal/core/models.py

```diff
@@ -414,7 +420,7 @@ class ServicePage(Page):
     api_fields = [
         APIField("icon_classes"),
         APIField("overview"),
-        APIField("head_of_service"),
+        APIField("head_of_service", serializer=AdministratorPageSerializer(source='*')),
```

Comment:
Remember we need the same functionality for the child pages of AdministratorIndexPage and CouncillorListPage

Reply:
Can you provide all API endpoints which I should update?

Reply:
core.CouncillorListPage.child_pages
    core.AdministrationIndexPage.child_pages
    core.CouncillorGroupPage.councillors
    core.ServicePage.head_of_service

Each of these are one or more PersonPage subclasses which have a profile image - from each of these pages. We need the thumbnail in the data for each of these pages.